flake.nix

```diff
@@ -19,6 +19,7 @@
     pre-commit-hooks.inputs.nixpkgs.follows = "nixpkgs";
     pre-commit-hooks.inputs.flake-utils.follows = "flake-utils";
     haskell-nix-utils.url = "github:TerrorJack/haskell-nix-utils";
+    jupyter-flake.url = "git+https://github.com/aleeusgr/monad-bayes?ref=notebooks";
```

Comment:
Why is there a reference to your fork?

Reply:
Huh - I just saw your comment above - let me try and ask the experts at tweag

Reply:
I just changed the url to tweag and it works for me (that's not how nix is supposed to work)

Reply:
I think it has to do with cached versions of stuff in nix store. after rebase the error seems to be gone.